src/Sportalytics.API.Feed/Controllers/FeedController.cs

```diff
@@ -0,0 +1,28 @@
+﻿using MediatR;
+using Microsoft.AspNetCore.Mvc;
+using Sportalytics.Application.Feed.Commands;
+using Sportalytics.Application.Feed.Queries;
+using Sportalytics.Domain.DTOs;
+
+namespace Sportalytics.API.Feed.Controllers;
+
+[Route("api/[controller]")]
+[ApiController]
+public sealed class FeedController(ISender sender) : ControllerBase
+{
+    [HttpPost]
+    public async Task<IActionResult> CreateSportEvent([FromBody] CreateSpotEventDto createSpotEventDto, CancellationToken cancellationToken)
+    {
+        var command = new CreateSportEventCommand(createSpotEventDto, cancellationToken);
+        var result = await sender.Send(command, cancellationToken);
```

Comment:
`SendAsync`

Reply:
“Send” is async call and don't have “SendAsync” name.